integrations/common/rest/src/main/java/io/helidon/integrations/common/rest/RestApi.java

```diff
@@ -276,16 +262,13 @@ Single<T> invokeOptional(Method method,
      * @param <T> type of the built {@link io.helidon.integrations.common.rest.RestApi}
      */
     abstract class Builder<B extends Builder<B, T>, T extends RestApi> implements io.helidon.common.Builder<B, T> {
-        private final WebClient.Builder webClientBuilder = WebClient.builder()
-                .followRedirects(true)
-                .keepAlive(true);
+        private final Http1ClientBuilder webClientBuilder = Http1Client.builder().followRedirect(true).keepAlive(true);
```

Comment:
Break on multiple lines, so that we only see the first line changed.

Reply:
Fixed.